recipes/nyxus/meta.yaml

```
@@ -0,0 +1,74 @@
+{% set name = "nyxus" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PolusAI/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e82f349f6ff481e017c3ff2060642e66e81dbda5910272fba1b3c5a4b5d9b2fd
+  patches:
+    - 0001-fix-filesystem-issue.patch
+    - 0002-Fix-CUDA-Kernel-Issue.patch
+
+build:
+  number: 0
+  skip: true  # [py<36]
+  missing_dso_whitelist:
+    - "*/nvcuda.dll"   # [win]
+
+requirements:
+  build:
+    - cmake
+    - make  # [not win]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
+    - git 
+
+  host:
+    - python
+    - setuptools
+    - pip
+    - z5py 
+    - libtiff 
+    - boost 
+    - nlohmann_json 
+    - blosc 
```

```suggestion
    - blosc  # [cuda_compiler_version == "None"]
```
It looks like blosc is unused in CUDA builds because the link inspector says there are no links to it.

---

This I believe only happens for CUDA 10.2. The build machine uses GCC 7.5 which fails the minimum GCC requirement for `xtensor`. There is a open issue in the `xtensor` Github about this. In this particular situation, we build Nyxus w/o zarr file support. CUDA 11.x uses blosc. So, should I use `# [cuda_compiler_version == "None" or cuda_compiler_version == "10.2.0"] ` ?

---

@carterbox : I updated meta.yaml and CMakeLists.txt so that it does not used bring in blosc in CUDA 10.2 build (where it is not used). I think the link inspector now does not find it anymore.

---

> So, should I use # [cuda_compiler_version != "10.2.0"] ?

Yes, something like that is fine. Could have also dropped builds before 11.